smarts/core/scenario.py

```diff
@@ -195,22 +187,20 @@ def variations_for_all_scenario_roots(
             ) in product(
                 np.roll(routes, roll_routes, 0),
                 np.roll(agent_missions, roll_agent_missions, 0),
-                np.roll(social_agents, roll_social_agents, 0),
+                np.roll(social_agent_infos_and_missions, roll_social_agents, 0),
                 np.roll(traffic_histories, roll_traffic_histories, 0),
             ):
+                if concrete_social_agents is None:
```

Comment:
referenced before assignment?

Reply:
No it is created from the `for` iteration.